lib/modules/dosomething/dosomething_fact_page/dosomething_fact_page.module

```diff
@@ -90,22 +89,29 @@ function dosomething_fact_page_preprocess_node(&$vars) {
 
     foreach ($template_vars as $key => $labels) {
       foreach ($labels as $label) {
-        $field = "field_{$label}";
-          if (isset($content[$field])) {
+        $field = $vars["field_{$label}"];
+
+        if (isset($field) && !empty($field)) {
+          if (array_key_exists('en', $field)) {
+            $field = $field['en'];
```

Comment:
Yep yep, I did something similar in this PR with what you're suggesting :) https://github.com/DoSomething/phoenix/commit/8ec376c56643d1d4196b1920587e36477a6bb5a3